client/my-sites/term-tree-selector/index.jsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import TermTreeSelectorTerms from './terms';
```

Comment:
I might suggest looking to see if we can eliminate `./terms` as a separate component altogether and merge into this one. I think it was a problem in `<PostSelector />` only because of the need to compile a query object and manage page in state.